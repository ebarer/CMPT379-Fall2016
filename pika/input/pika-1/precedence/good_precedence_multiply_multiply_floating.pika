exec {

  print 1.1 * 9.8 * 7.9, _n_.
  print 9.8 * 2.8 * 3.7, _n_.
  print 4.4 * 6.6 * 8.2, _n_.
  print 8.4 * 8.4 * 3.7, _n_.
  print 2.1 * 4.6 * 3.5, _n_.
  print 7.3 * 7.2 * 1.9, _n_.
  print 8.7 * 3.3 * 6.0, _n_.
  print 8.6 * 3.3 * 4.4, _n_.
  print 4.6 * 6.8 * 4.6, _n_.
  print 1.4 * 1.0 * 4.4, _n_.
  print 6.8 * 9.5 * 7.9, _n_.
  print 6.7 * 3.2 * 8.7, _n_.

  print 5.6 * 7.9 * 5.0, _n_.
  print 5.5 * 5.9 * 3.9, _n_.
  print 3.2 * 9.5 * 3.9, _n_.
  print 1.1 * 4.8 * 2.2, _n_.
  print 6.3 * 6.4 * 6.7, _n_.
  print 6.0 * 2.6 * 1.3, _n_.
  print 2.5 * 3.5 * 3.2, _n_.
  print 3.7 * 8.5 * 9.9, _n_.
  print 1.6 * 9.1 * 5.0, _n_.
  print 2.4 * 9.8 * 2.9, _n_.
  print 9.2 * 9.7 * 5.6, _n_.
  print 2.4 * 5.6 * 1.8, _n_.

  print 7.7 * 5.8 * 2.0 * 3.4, _n_.
  print 2.8 * 3.3 * 3.8 * 1.9, _n_.
  print 7.7 * 2.8 * 8.3 * 7.9, _n_.
  print 1.7 * 6.3 * 9.8 * 6.6, _n_.
  print 1.1 * 6.1 * 9.2 * 4.3, _n_.
  print 5.2 * 5.6 * 3.0 * 5.5, _n_.
  print 7.0 * 9.6 * 6.5 * 5.4, _n_.
  print 5.4 * 4.0 * 4.4 * 8.1, _n_.
  print 4.3 * 4.1 * 8.8 * 4.4, _n_.
  print 8.7 * 4.5 * 8.4 * 4.7, _n_.
  print 7.6 * 4.9 * 8.5 * 6.2, _n_.
  print 6.2 * 7.4 * 4.2 * 4.2, _n_.

  print 4.5 * 7.7 * 2.4 * 6.3, _n_.
  print 4.0 * 6.5 * 5.5 * 6.5, _n_.
  print 4.6 * 9.2 * 6.9 * 6.5, _n_.
  print 2.5 * 7.3 * 6.0 * 4.1, _n_.
  print 4.6 * 6.0 * 6.4 * 9.6, _n_.
  print 5.9 * 1.2 * 7.6 * 7.2, _n_.
  print 9.6 * 9.7 * 2.3 * 1.7, _n_.
  print 7.9 * 1.0 * 4.0 * 2.7, _n_.
  print 3.0 * 3.6 * 9.5 * 1.3, _n_.
  print 9.4 * 3.8 * 8.1 * 4.0, _n_.
  print 3.2 * 3.5 * 3.0 * 7.5, _n_.
  print 2.7 * 8.0 * 5.5 * 3.8, _n_.

}

