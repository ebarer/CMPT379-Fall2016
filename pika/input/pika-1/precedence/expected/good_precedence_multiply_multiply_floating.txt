85.162
101.528
238.128
261.072
33.81
99.864
172.26
124.872
143.888
6.16
510.34
186.528
221.2
126.555
118.56
11.616
270.144
20.28
28
311.355
72.8
68.208
499.744
24.192
303.688
66.7128
1413.69
692.723
265.448
480.48
2358.72
769.824
682.634
1545.64
1962.55
809.323
523.908
929.5
1898.05
448.95
1695.74
387.418
364.099
85.32
133.38
1157.33
252
451.44
